shop/src/algosigner.config.js

```diff
@@ -1,2 +0,0 @@
-// algosigner ledger name [ MainNet | TestNet | <private_net_display_name> ]
```

Comment:
Is this file supposed to be deleted?

Reply:
yes, it is specific to algosigner, but since we now have other wallets also, I created another file `config.js`